src/utils/types.utils.ts

```diff
@@ -0,0 +1,34 @@
+export enum VulDatabase {
+  NVD = 1,
+  Sonatype,
+  /*Add supported DBs here */
+}
+
+export interface Query {
+  database: VulDatabase;
+  method: string;
+  url: string;
```

Comment:
very humble suggestion but maybe it would be easier to have the `VulDatabase` enum contain the urls for each database. From the test cases I see we always hit the same url for NVD and the same one for sonatype.

Maybe we can switch it so the value for url doesn't need to be known and passed in by the higher level components that use `sendQuery`

Reply:
Discussed two options:
1. make QueryFacade more general and return raw response
2. make Query more specific - couple URL to database type

going with number2 for now!